src/blocks/blocks.service.ts

```diff
@@ -378,7 +378,7 @@ export class BlocksService {
     }
 
     return {
-      averageBlockTimeMs: dateMetricsResponse[0].average_block_time_ms,
+      averageBlockTimeMs: Number(dateMetricsResponse[0].average_block_time_ms),
```

Comment:
Not sure why this wasn't needed before, exactly. But the unit test started failing due to this being an object (and it _was_ indeed an object before being cast according to `typeof`)